### DISCUSSION TITLE
Are you planning bumping preview2?

### DISCUSSION THREAD

**Discussion by danipen** [wieslawsoltes/Svg.Skia#126] [Q&A] [answered]
Hey @wieslawsoltes hi! 

Are you planning to bump a new nuget upstreaming Avalonia `11.0.0-preview2`?

**Comment by wieslawsoltes** [comment]
Yes

**Reply by danipen** [reply]
do you have an estimated date?

**Reply by wieslawsoltes** [reply]
> do you have an estimated date?

Should be on NuGet in few minutes

**Comment by wieslawsoltes** [comment, marked as answer]
https://github.com/wieslawsoltes/Svg.Skia/releases/tag/0.6.0-preview2

**Comment by danipen** [comment]
OMG lightning fast! Thanks!!
